src/monorepo-shipit/src/RepoGitFake.js

```diff
@@ -20,7 +19,7 @@ export default class RepoGitFake extends RepoGit {
     new ShellCommand(testRepoPath, 'git', 'init').runSynchronously();
     const username = 'adeira-shipit-tests';
```

Comment:
Perhaps this username could be inlined as well now?

Reply:
Done